api/src/provisioning_start.ts

```diff
@@ -0,0 +1,111 @@
+import { FastifyInstance } from "fastify";
+import { VError } from "verror";
+import { AuthenticatedRequest } from "./httpd/lib";
+import { toHttpError } from "./http_errors";
+import * as NotAuthenticated from "./http_errors/not_authenticated";
+import { Ctx } from "./lib/ctx";
+import * as Result from "./result";
+import { ServiceUser } from "./service/domain/organization/service_user";
+import Joi = require("joi");
+
+interface RequestBodyV1 {
+  apiVersion: "1.0";
+  data: {};
+}
+
+const requestBodyV1Schema = Joi.object({
+  apiVersion: Joi.valid("1.0").required(),
+  data: Joi.object().empty().required(),
+});
+
```

Comment:
yes stefan, sending an empty object in a post call is best practice, if no paramater are needed